src/core/rewrite.ml

```diff
@@ -161,22 +153,22 @@ let _ =
     let prod = _Prod term_T_a (Bindlib.bind_var x prod) in
     Bindlib.unbox (_Prod term_U (Bindlib.bind_var a prod))
   in
-  Builtin.Check.register_expected_type "eqind" expected_eqind
-
-(** [get_eq_data cfg a] extra data from an equality type [a]. It consists of a
-    triple containing the type in which equality is used and the equated terms
-    (LHS and RHS). *)
-let get_eq_data : popt -> eq_config -> term -> term * term * term =
-  fun pos cfg t ->
-  match get_args t with
+  register_builtin "eqind" expected_eqind_type
+
+(** [get_eq_data pos cfg a] extra data from an equality type [a]. It
+   consists of a triple containing the type in which equality is used and the
+   equated terms (LHS and RHS). *)
```

Comment:
Please, indent comments correctly.

Reply:
See my previous comment. Thanks to fix indentation yourself. I don't have the time to do this.